### PR TITLE
Procedural Changes

### PR DESCRIPTION
**Pull request by joinimran**
An unnecessary step was added in the procedure and has been removed. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8408

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/microsoft-defender-atp/mac-jamfpro-policies.md [hunk 1]
```diff
@@ -750,18 +750,14 @@ Follow the instructions on [Schedule scans with Microsoft Defender for Endpoint
 
     ![Image of configuration settings](images/990742cd9a15ca9fdd37c9f695d1b9f4.png)
 
-4. Navigate to **Advanced Computer Searches**.
-    
-    ![A screenshot of a social media post Description automatically generated](images/95313facfdd5e1ea361981e0a2478fec.png)
-
-5. Select **Computer Management**. 
+4. Select your computer and click the gear icon on the top, select **Computer Management**
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/mac-jamfpro-policies.md:753]:
```suggestion
4. Select your computer and click the gear icon at the top, then select **Computer Management**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - it seems that a number of images also needs to be removed now. @joinimran could you please check and confirm?

**Comment by illfated** [comment]
Rich diff view: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8901/files?short_path=168119a (in case it helps)

**Comment by joinimran** [comment]
There was one image that needs to be removed and was removed. So we are good with this.

**Review by illfated — APPROVED** [review]
Rich Diff view: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8901/files?short_path=168119a#diff-168119a330b6d49028183467ea1cb90824f5a3c3e27b6243b6ad522447db338d

Looks good to me.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Pull request merged**